docs/snippets/react/component-story-with-query.js.mdx

```diff
@@ -0,0 +1,23 @@
+```js
+// my-component-with-query.stories.js
+
+import MyComponentThatHasAQuery, { MyQuery} from '../component-that-has-a-query';
+
+export const LoggedOut = () => <MyComponentThatHasAQuery />;
```

Comment:
Let's use a template.

Reply:
Changed the snippet to include a template with the latest commit.

Let me know if you need anything else.